io/DerivativeWriter.cpp

```diff
@@ -161,9 +160,17 @@ void DerivativeWriter::write(const PointViewPtr data)
                     continue;
                 Matrix3d block = paddedDSM.block(r-1, c-1, 3, 3);
                 if (to.m_type == SLOPE_D8)
+                {
```

Comment:
I guess I thought you meant that we had it defined within PDAL already. I also always forget about the `atan` and `acos` "tricks" to obtaining PI.

I'll probably just make a percent slope to degrees slope function to document what's going on.